aiida/tools/importexport/common/zip_folder.py

```diff
@@ -0,0 +1,152 @@
+# -*- coding: utf-8 -*-
+###########################################################################
+# Copyright (c), The AiiDA team. All rights reserved.                     #
+# This file is part of the AiiDA code.                                    #
+#                                                                         #
+# The code is hosted on GitHub at https://github.com/aiidateam/aiida-core #
+# For further information on the license, see the LICENSE.txt file        #
+# For further information please visit http://www.aiida.net               #
+###########################################################################
+"""Export a zip-file."""
```

Comment:
Could you add some explanation why this is necessary for zip but not for tar?

Reply:
FYI I simply moved this file from its original location `aiida/tools/importexport/dbexport/zip.py`, without change. But no problem, I can address the review comments here and below

Also, I left the original file there with a deprecation warning, but can fully remove if you think this is not necessary for back-compatibility.

Reply:
I personally don't expect that `aiida/tools/importexport/dbexport/zip.py` was used directly by plugins, so for me it's ok to remove.